tools/create_index/index_generator.js

```diff
@@ -1,36 +1,51 @@
-var fs = require('fs')
+const fs = require('node:fs')
+const satisfies = require('semver/functions/satisfies')
 
 // We will fill in this object with our data
-var vuln = {}
+let vuln = {}
 
 const createIndex = function (vulnDirectoryPath) {
   const files = fs.readdirSync(vulnDirectoryPath)
   getVulnDirectoryContents(files, vulnDirectoryPath)
   writeIndex(vuln, vulnDirectoryPath)
 }
 
+const supportedVersions = ['14.0.0', '16.0.0', '18.0.0', '19.0.0']
```

Comment:
Is there some way to avoid hard coding this?

Reply:
Easily, I don't think so.  @nodejs/releasers do we have an API with this info?

Reply:
I'll merge it. In case someone has a better idea, I can create a follow-up PR.

Reply:
> Easily, I don't think so. @nodejs/releasers do we have an API with this info?

https://github.com/pkgjs/nv with the `supported` alias.